crates/host/src/module.rs

```diff
@@ -292,14 +330,7 @@ impl SerializedModuleCache {
                 // Each module needs to be compiled with a new engine because
                 // of middleware like metering. Middleware is compiled into the
                 // module once and available in all instances created from it.
-                let mut compiler_engine = (self.make_compiler_engine)();
-                // Workaround for invalid memory access on iOS.
-                // https://github.com/holochain/holochain/issues/3096
-                compiler_engine.set_tunables(BaseTunables {
-                    static_memory_bound: 0x4000.into(),
-                    static_memory_offset_guard_size: 0x1_0000,
-                    dynamic_memory_offset_guard_size: 0x1_0000,
-                });
+                let compiler_engine = (self.make_engine)();
```

Comment:
```suggestion
                let compiler_engine = self.make_engine();
```

right?

Reply:
fixed

Reply:
reverted -- `make_engine` isn't a function it's a function pointer